keyboards/utd80/utd80.c

```diff
@@ -0,0 +1,41 @@
+#include "utd80.h"
+
+
+void matrix_init_kb(void) {
+  // Keyboard start-up code goes here
+  // Runs once when the firmware starts up
+  matrix_init_user();
+  led_init_ports();
+};
+
+void matrix_scan_kb(void) {
+  // Looping keyboard code goes here
+  // This runs every cycle (a lot)
+  matrix_scan_user();
+};
+
+void led_init_ports(void) {
+    DDRB |= (1<<5) | (1<<6); // OUT

```

Comment:
```suggestion
   setPinOutput(B5);
   setPinOutput(B6);
```

you'll need to include `#include 'quantum.h'` for this.

Reply:
> you'll need to include `#include 'quantum.h'` for this.

I included 'quantum.h' in the 'utd80.h'.